### DISCUSSION TITLE
Avoid `css-overflow-wrap-anywhere` on tables!

### DISCUSSION THREAD

**Discussion by SharakPL** [community/community#79109] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

`css-overflow-wrap-anywhere` class on markdown tables sets `overflow-wrap: anywhere` and it breaks tables with long content:

![obraz](https://github.com/community/community/assets/5007456/b9ebaf52-0e37-41de-9daf-c895341857f3)

It should be set to `break-word`. Expected rendering:

![obraz](https://github.com/community/community/assets/5007456/3b6ea8cc-ccb5-4500-8e61-2c0e182293cd)

**Comment by isker** [comment]
I see this class added to PR review comments but not to PR comments.  Seems bad in any case.

**Comment by tienanh109** [comment]


**Reply by ColeDCrawford** [reply]
This is coming from Github, so it's not possible to modify this directly, right? In my case, I have an Action that adds a coverage report as a comment on a commit. I don't think the Action is adding this class to the table, I think it's how Github is formatting the comment:

<img width="782" alt="Screenshot 2024-09-04 at 2 18 47 PM" src="https://github.com/user-attachments/assets/0d4d43f8-06a2-46a0-9acc-bd02e337754a">

**Reply by isker** [reply]
Yes, it’s a nonsense AI response.
